src/frontends/paddle/src/frontend.cpp

```diff
@@ -135,22 +135,32 @@ bool normalize_framework_node(const std::shared_ptr<FrameworkNode>& node,
     return true;
 }
 
-std::istream* variant_to_stream_ptr(const ov::Any& variant, std::ifstream& ext_stream) {
+OPENVINO_SUPPRESS_DEPRECATED_START
+std::istream* variant_to_stream_ptr(const ov::Any& variant,
+                                    std::ifstream& ext_ifstream,
+                                    std::stringstream& ext_sstream) {
```

Comment:
Can we merge `std::ifstream& ext_ifstream` and `std::stringstream& ext_sstream` into together, because we can see that each branch only uses one of them.

Reply:
> Can we merge `std::ifstream& ext_ifstream` and `std::stringstream& ext_sstream` into together, because we can see that each branch only uses one of them.

@riverlijunjie I simplify the function variant_to_stream_ptr. Please review when you are free.